### DISCUSSION TITLE
Swiper.js slows down on autoplay between slides

### DISCUSSION THREAD

**Discussion by StrongereCommerce** [nolimits4web/swiper#5497] [Q&A]
Hi all,

My first question here on Github! Hopefully someone can help.

I created an autoplay slider with swiper.js and although it keeps sliding, it always stops between each slide. How do I get it to continue smoothly?
here is a demo code: https://codepen.io/swissmedia/pen/xxPaZqj

**Comment by Arhell** [comment]
Check
https://codepen.io/hammezg/pen/gEpGdq
https://stackoverflow.com/questions/64853149/how-to-create-a-ticker-mode-slider-with-swiper-js

**Reply by StrongereCommerce** [reply]
Hi, Thanks for your reply. Unfortunately, that codepen also has the same issue. It keeps giving a tiny stop between slides.
